src/react-queries/queries/thornode.ts

```diff
@@ -65,11 +65,12 @@ export const thornode = createQueryKeys('thornode', {
           // Get all inbound addresses
           (
             await thorService.get<InboundAddressResponse[]>(
-              `${thornodeUrl}/lcd/thorchain/inbound_addresses`,
+              `${thornodeUrl}/thorchain/inbound_addresses`,
             )
           ).andThen(({ data: inboundAddresses }) => {
             // Exclude halted
-            const activeInboundAddresses = inboundAddresses.filter(a => !a.halted)
+            // TODO(gomes): revert me, shit's halted
+            const activeInboundAddresses = inboundAddresses
```

Comment:
And yeah, obviously revert me before we land this - keeping it for the sake of testing

Reply:
8689a0061